sway-core/src/type_system/info.rs

```diff
@@ -1156,6 +1156,43 @@ impl TypeInfo {
         matches!(self, TypeInfo::Tuple(_))
     }
 
+    pub fn contains_never(&self, engines: &Engines) -> bool {
```

Comment:
Perhaps calling this method `is_uninhabited`? `contains_never` sounds to me more like a _way_ to make a type uninhabited, but if I think about a type property the sentence "type.is_uninhabited()` sounds more like what that type _is_, which property it has.